--- external/Elements/HtmlListItem.php
+++ repo/Elements/HtmlListItem.php
@@ -9,11 +9,8 @@
 class HtmlListItem extends HtmlElement
 {
 	//Held within either an Ordered List or Unordered List
-	function __construct( $data )
+	function __construct( HtmlElementInterface $data )
 	{
-		if (!($data instanceof HtmlElementInterface)) {
-			$data = new HtmlString((string)$data);
-		}
 		parent::__construct( $data );
 		$this->tag = "li";
 	}