--- external/Elements/HtmlInserted.php
+++ repo/Elements/HtmlInserted.php
@@ -3,13 +3,12 @@
 namespace Ksfraser\HTML\Elements;
 
 use Ksfraser\HTML\HtmlElementInterface;
-use Ksfraser\HTML\Formatting\HtmlFormatting;
 
 class HtmlInserted extends HtmlFormatting
 {
-	       function __construct( HtmlElementInterface $data )
-	       {
-		       parent::__construct( $data );
-		       $this->tag = "ins";
-	       }
+	function __construct( $data )
+	{
+		parent::__construct( $data );
+		$this->tag = "ins";
+	}
 }